SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetConnection.h

```diff
@@ -64,9 +64,6 @@ class SPATIALGDK_API USpatialNetConnection : public UIpConnection
 	// the PlayerController as a partition entity for the client worker.
 	Worker_EntityId ConnectionClientWorkerSystemEntityId;
 
-	class FTimerManager* TimerManager;
```

Comment:
Is this a separate PR? This looks like you found and are cleaning up some dead code? Please place this in a separate commit/PR.

Reply:
Yes, dead code by some mistake I guess.

Reply:
Seems like this was left over after the client connection manager PR. I think it's fine to clean this up as a drive by.

Reply:
yep- thanks for clearing this up